aten/src/ATen/cpu/vec/vec_mask.h

```diff
@@ -134,6 +134,14 @@ class VecMask {
     return VectorizedN<T, N>(VectorizedN<T, N>::loadu(mask));
   }
 
+  void store(bool* b, int count) {
+    TORCH_CHECK(count == Vectorized<T>::size() * N, "Expect same number of elements in VecMask::store.");
+    TORCH_CHECK(count <= Vectorized<bool>::size(), "Expect the number is less or equal to Vectorized<bool>.");
```

Comment:
No need to explicit assert here? It should be caller's responsibility to guarantee the count is valid?

Reply:
Sure and removed.